### PR TITLE
Update ext.html.erb

### PR DESCRIPTION
**Pull request by packtpartner**
Added a new video listing

### CODE REVIEW

File: app/views/doc/ext.html.erb [hunk 1]
```diff
@@ -88,6 +88,12 @@
           <p class='description'>
             Your Guide to deliver great code using TFS and Git.
           </p>
+        </li>
+        <li>
+          <h4><a href="https://www.packtpub.com/networking-and-servers/complete-git-and-github-masterclass-beginner-git-expert-video">Complete Git and GitHub Masterclass : Beginner to Git Expert [Video]</a></h4>
+          <p class='description'>
+            Most comprehensive, STEP by STEP Git & GitHub tutorial with great examples; Upcoming: Git Workflow, BitBucket and more!
```
Comment (pedrorijo91) [inline app/views/doc/ext.html.erb]: I'm not a big fan of this kind of propaganda "most comprehensive step by step tutorial". 
It's a step by step git & GitHub tutorial with examples, it's not the best tutorial in the world.

Even the 'upcoming'  section is a bit unnecessary and deprecated-prone

Comment (peff) [reply app/views/doc/ext.html.erb]: Agreed. Let's try to keep these descriptions as neutral as possible.

Comment (pedrorijo91) [reply app/views/doc/ext.html.erb]: @packtpartner are you interested in updating this description to be more neutral or can this PR be closed?

Comment (packtpartner) [reply app/views/doc/ext.html.erb]: Yes, the description now is good to go. Thank you

**Pull request closed without merging**